activemq-rar/pom.xml

```diff
@@ -114,7 +114,7 @@
           <artifactId>jakarta-regexp</artifactId>
         </exclusion>
         <exclusion>
-          <groupId>javax.jmdns</groupId>
+          <groupId>org.jmdns</groupId>
```

Comment:
I think the exclusion here is still using the javax.jmdns groupId.

Reply:
@jbonofre the dependency tree looks correct to me. This is from a full build off the PR branch.

```
[INFO] --- maven-dependency-plugin:3.3.0:tree (default-cli) @ activemq-rar ---
[INFO] org.apache.activemq:activemq-rar:rar:5.18.0-SNAPSHOT
[INFO] +- org.apache.activemq:activemq-spring:jar:5.18.0-SNAPSHOT:compile
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.36:compile
[INFO] |  +- org.apache.activemq:activemq-broker:jar:5.18.0-SNAPSHOT:compile
[INFO] |  |  +- org.apache.activemq:activemq-client:jar:5.18.0-SNAPSHOT:compile
[INFO] |  |  +- org.apache.activemq:activemq-openwire-legacy:jar:5.18.0-SNAPSHOT:compile
[INFO] |  |  +- javax.annotation:javax.annotation-api:jar:1.3.2:compile
[INFO] |  |  \- com.fasterxml.jackson.core:jackson-databind:jar:2.14.2:compile
[INFO] |  |     +- com.fasterxml.jackson.core:jackson-annotations:jar:2.14.2:compile
[INFO] |  |     \- com.fasterxml.jackson.core:jackson-core:jar:2.14.2:compile
[INFO] |  \- org.apache.activemq:activemq-pool:jar:5.18.0-SNAPSHOT:compile
[INFO] |     \- org.apache.activemq:activemq-jms-pool:jar:5.18.0-SNAPSHOT:compile
[INFO] +- org.apache.activemq:activemq-ra:jar:5.18.0-SNAPSHOT:compile
[INFO] |  \- org.apache.activemq:activemq-kahadb-store:jar:5.18.0-SNAPSHOT:compile
[INFO] +- org.apache.xbean:xbean-spring:jar:4.22:compile
[INFO] |  \- commons-logging:commons-logging:jar:1.2:compile
[INFO] +- org.springframework:spring-core:jar:5.3.25:compile
[INFO] |  \- org.springframework:spring-jcl:jar:5.3.25:compile
[INFO] +- org.springframework:spring-context:jar:5.3.25:compile
[INFO] |  +- org.springframework:spring-aop:jar:5.3.25:compile
[INFO] |  +- org.springframework:spring-beans:jar:5.3.25:compile
[INFO] |  \- org.springframework:spring-expression:jar:5.3.25:compile
[INFO] +- org.apache.commons:commons-pool2:jar:2.11.1:compile
[INFO] +- org.apache.activemq:activemq-amqp:jar:5.18.0-SNAPSHOT:compile
[INFO] |  +- org.apache.qpid:proton-j:jar:0.33.10:compile
[INFO] |  \- org.fusesource.hawtbuf:hawtbuf:jar:1.11:compile
[INFO] +- org.apache.activemq:activemq-mqtt:jar:5.18.0-SNAPSHOT:compile
[INFO] |  +- org.apache.activemq.protobuf:activemq-protobuf:jar:1.1:compile
[INFO] |  +- org.fusesource.mqtt-client:mqtt-client:jar:1.16:compile
[INFO] |  \- org.fusesource.hawtdispatch:hawtdispatch-transport:jar:1.22:compile
[INFO] |     \- org.fusesource.hawtdispatch:hawtdispatch:jar:1.22:compile
[INFO] +- org.apache.logging.log4j:log4j-core:jar:2.19.0:compile
[INFO] |  \- org.apache.logging.log4j:log4j-api:jar:2.19.0:compile
[INFO] +- org.apache.activemq:activemq-jaas:jar:5.18.0-SNAPSHOT:compile
[INFO] +- org.apache.logging.log4j:log4j-slf4j-impl:jar:2.19.0:runtime
[INFO] +- org.apache.activemq:activemq-all:jar:5.18.0-SNAPSHOT:test
[INFO] \- junit:junit:jar:4.13.2:test
[INFO]    \- org.hamcrest:hamcrest-core:jar:1.3:test
```

Reply:
Ah yes, correct as jmdns is coming from activemq-spring that uses the new coordinates.